message/nimessage.yml

```
@@ -47,6 +58,21 @@ parameters:
     description: Unique session ID
     type: string
     required: true
+  CreateSessionRequest:
+    in: body
+    name: createSessionRequest
+    description: Request to create a new session
+    schema:
+      title: Create Session Request
+      description: reates a message session with the server to send and receive messages.
+      type: object
+      properties:
+        workspace:
+          type: string
+          description: The workspace used by this session
+          x-example: 8472777d-8e2f-4c42-bceb-7901ede23085
```

```suggestion
          example: 8472777d-8e2f-4c42-bceb-7901ede23085
```
Models can have normal examples.